W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/add-request-response-details.adoc

  /{ID}:
    get:
--

> . Add an `!include` directive and press Ctrl+Spacebar open the autocomplete options.

Author: ```suggestion
. Add an `!include` directive, and press Ctrl+Spacebar to open the autocomplete options.
```

Create a new `example:` element for the `response` element for your `/flights` resource:

> . Add a new resource at the same level as your `type` response by pressing Ctrl+Spacebar (Windows) or Cmd+Spacebar (Mac) to open the autocomplete options and selecting `examples`.

Author: ```suggestion
. Add a new resource at the same level as your `type` response by pressing Ctrl+Spacebar (Windows) or Cmd+Spacebar (Mac) to open the autocomplete options, and select `examples`.
```

Additionally, you can define the type of response of your `post` method for your `/flights` resource:

. Add a new line under your `/flights:post` method and indent it at the same level as your `body:` element.

> . Press Ctrl+Spacebar (Mac) to open the autocomplete options to create a 201 response:

Author: ```suggestion
. Press Ctrl+Spacebar (Mac) to open the autocomplete options and generate a 201 response:
```
